Formula/newrelic-infra-agent.rb

```diff
@@ -0,0 +1,42 @@
+class NewrelicInfraAgent < Formula
+  desc "New Relic infrastructure agent"
+  homepage "https://github.com/newrelic/infrastructure-agent"
+  url "https://github.com/newrelic/infrastructure-agent/archive/refs/tags/1.20.3.tar.gz"
+  sha256 "3a29b19541e6f32fb87196a7833d08a9aed41b8ff62ef3a72326236d262cb033"
+  license "Apache-2.0"
+  head "https://github.com/newrelic/infrastructure-agent.git"
+
+  depends_on "go" => :build
+
+  def install
+    goarch = Hardware::CPU.arm? ? "arm64" : "amd64"
+    cgo = Hardware::CPU.arm? ? "0" : "1"
+    ENV["VERSION"] = "1.20.3"
```

Comment:
```suggestion
    ENV["VERSION"] = version
```
Did this not work? If it gave you problems, we can do
```suggestion
    ENV["VERSION"] = version.to_s
```

Reply:
Will this version be fetched from archive url?

Reply:
Yes.

Reply:
fixed